clusters/app.ci/supplemental-ci-images/html-proofer_is.yaml

```diff
@@ -22,6 +22,36 @@ spec:
     to:
       kind: ImageStreamTag
       name: html-proofer:bash-3.19.0
+    imageLabels:
```

Comment:
I meant the label on the BC, not on the image.
The goal is that we can filter out easily in the future with the label